common/anomalies/hfe_anomaly.txt

```diff
@@ -66,15 +67,15 @@ hfe_anomaly_criopods = {
 			years_passed > 30
 		}
 		modifier = {
-			factor = 2
+			factor = 5
```

Comment:
Values to hight, animalies is not intended to show up in every game

Reply:
I raised the multiples for the LATE GAME, not the Early Game. After a hundred twenty years, almost everything is surveyed.

Reply:
However, per your request, I will revise it downwards